src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -51,6 +51,7 @@ var strings = {
     'close': django.pgettext('widget', 'Close'),
     'continue': django.pgettext('widget', 'Continue'),
     'variations': django.pgettext('widget', 'See variations'),
+    'hide_variations': django.pgettext('widget', 'Hide variations'),
```

Comment:
As discussed change to match pretix-presale:
```suggestion
    'variations': django.pgettext('widget', 'Show variants'),
    'hide_variations': django.pgettext('widget', 'Hide variants'),
```